build-wabac.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+WABAC_SW_URL=https://cdn.jsdelivr.net/npm/@webrecorder/wabac@2.21.4/dist/sw.js
+
+wget "$WABAC_SW_URL" -O ./pywb/static/wabacWorker.js
```

Comment:
```suggestion
wget "$WABAC_SW_URL" -O ./pywb/static/wabacSW.js
```
For my own sanity, would be better to call it wabacSW.js as it's usually sw.js - it can also run as a web worker for loading and not a service worker, but we aren't doing that.